src/popupmenu.c

```diff
@@ -115,6 +115,8 @@ pum_display(
     do
     {
 	def_width = p_pw;
+	if (p_pmw > 0 && def_width > p_pmw)
```

Comment:
then we can save the test for `p_pmw > 0` test here and you can use `def_width = MAX(p_pmw, p_pw);`

Reply:
emm i don't think so.. 0 means no maximum limit. pmw is the maximum value. Even if it is smaller than pumwidth, pumwidth still uses pmw.